pkg/controller/common/nodenetwork.go

```diff
@@ -17,6 +17,7 @@ const (
 	Namespace        = "harvester-system"
 	KeyNodeName      = "NODENAME"
 	KeyLabelNodeName = "network.harvester.cattle.io/nodename"
+	initStatusMsg    = "initialize"
```

Comment:
Is this what we show in the UI? We should make it more readable.
showing a message like this could be confusing: `node1: initialize`

Reply:
Change message as `initializing network configuration` according to what suggested by guangbo.